lib/utils/constants.json

```diff
@@ -222,5 +222,10 @@
   },
   "MAX_SIMRINGS": 10,
   "BONG_TONE": "tone_stream://v=-7;%(100,0,941.0,1477.0);v=-7;>=2;+=.1;%(1400,0,350,440)",
-  "FS_UUID_SET_NAME": "fsUUIDs"
+  "FS_UUID_SET_NAME": "fsUUIDs",
+  "SystemStatusEvents" : {
```

Comment:
I would rather this be named "SystemState".  Also, we need an additional state "GracefulShutdownInProgress".  That needs to be triggered when the cluster scales in -- see utils/aws-sns-lifecycle

Reply:
@davehorton added GracefulShutdownInProgress event